### PR TITLE
Added a blog post for the survey

### PR DESCRIPTION
**Pull request by karthik**
Can you please a look and see if it reads ok? It's mostly to make sure that non Twitter users find this and respond.

### CODE REVIEW

**Comment by sckott** [comment]
Can do

File: _posts/2016-01-04-researcher-survey.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+layout: post
+title: Help us prioritize what to build in 2016
+date: 2016-01-04
+authors:
+  - name: Karthik Ram
+tags:
+- survey
+---
+
+We've got a big year ahead of us as we work towards expanding our team and organizing various events and activities. We remain committed to supporting and expanding the landscape of open source tools that are available to researchers. While much of our focus has been around making it easier to access various data repositories, we are keen on improving other parts of the research pipeline, including data munging, documentation and sharing. To help us prioritize our development activities and better understand the needs of researchers, we would really appreciate 5-12 minutes of your time to complete a survey:
```
Comment (sckott) [inline _posts/2016-01-04-researcher-survey.md]: what kind of "sharing"?

Comment (karthik) [inline _posts/2016-01-04-researcher-survey.md]: data? Should I rephrase that sentence? (the munging, documentation and sharing all refer to data)

Comment (sckott) [inline _posts/2016-01-04-researcher-survey.md]: ah, yeah, i guess it makes sense if they all follow "data"

File: _posts/2016-01-04-researcher-survey.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+<center><h3><a href="http://ropensci.org/survey">ropensci.org/survey</a></h3></center>
+
+The survey is not just for R users (we are keen on hearing from Python and Julia users and also folks that don't use these three languages). Please encourage your colleagues to participate. We'll summarize responses in a future post.
```
Comment (sckott) [inline _posts/2016-01-04-researcher-survey.md]: maybe not have the 2nd part of this sentence parenthetic, e.g

> The survey is not just for R users - we are keen on hearing from Python and Julia users and also folks that don't use these three languages.

**Comment by karthik** [comment]
Thanks Scott!

**Pull request merged**